pkg/local_object_storage/pilorama/forest.go

```diff
@@ -183,13 +184,21 @@ func (f *memoryForest) TreeGetOpLog(cid cidSDK.ID, treeID string, height uint64)
 
 // TreeDrop implements the pilorama.Forest interface.
 func (f *memoryForest) TreeDrop(cid cidSDK.ID, treeID string) error {
-	fullID := cid.String() + "/" + treeID
-	_, ok := f.treeMap[fullID]
-	if !ok {
-		return ErrTreeNotFound
+	cidStr := cid.String()
+	if treeID == "" {
+		for k := range f.treeMap {
+			if bytes.HasPrefix([]byte(k), []byte(cidStr)) {
```

Comment:
We can use same functionality from `strings` package.

Reply:
Thanks, fixed.